src/wallet/crypter.cpp

```diff
@@ -213,7 +213,7 @@ bool CCryptoKeyStore::Lock()
 bool CCryptoKeyStore::Unlock(const CKeyingMaterial& vMasterKeyIn)
```

Comment:
In `CryptoKeyStore::Lock()` at line 201, the test of `!SetCrypted()` should be inside the lock (as it is for `Unlock()`).

Reply:
Yes, I think this fixes a potential race condition.